amr-wind/turbulence/LES/OneEqKsgs.cpp

```diff
@@ -206,6 +210,47 @@ void OneEqKsgsM84<Transport>::update_scalar_diff(
     }
 }
 
+template <typename Transport>
+void OneEqKsgsM84<Transport>::post_advance_work() {
+
+    BL_PROFILE("amr-wind::" + this->identifier() + "::post_advance_work");
+
+    if ( m_hybrid_rl )   {
+        // Update sdr field based on sfs ke
+
```

Comment:
```suggestion
    if (!m_hybrid_rl) return; 
    
    BL_PROFILE("amr-wind::" + this->identifier() + "::post_advance_work");
```

Reply:
Fixed in 8ecbe368e5e6815e45ca2be481488572dfd0ea7f